File: lab2/week3/week3case.ts
class Person  {
    // 成员变量
    firstName: string;
    public lastName: string
    public age: number;
    
    // 构造函数
    constructor( firstName: string, lastName: string, age: number) {
      this.firstName = firstName;
      this.lastName = lastName;
      this.age = age;
    }
  
  
    // 私有方法
    private getBirthYear( currentYear:number): number {
      return  currentYear - this.age;
    }
  }
  

File: lab2/week3/week3case.java
public class Person {
    // 成员变量
    String firstName;
    public String lastName;
    public int age;

    // 构造函数
    public Person(String firstName, String lastName, int age) {
        this.firstName = firstName;
        this.lastName = lastName;
        this.age = age;
    }

    // 私有方法
    private int getBirthYear(int currentYear) {
        return currentYear - this.age;
    }
}